### PR TITLE
[BACKLOG-40517] - view project VFS connections

### PR DESCRIPTION
**Pull request by abryant-hv**

### CODE REVIEW

File: plugins/connections/ui/src/main/java/org/pentaho/di/connections/ui/tree/ConnectionFolderProvider.java [hunk 1]
```diff
@@ -71,4 +111,59 @@ public void create( AbstractMeta meta, TreeNode parent ) {
     refresh( meta, createTreeNode( parent, getTitle(), getTreeImage() ), null );
 
   }
+
+  // this could become common for all the types.
+  enum LEVEL {
+    PROJECT( "Project" ),
```
Comment (tgf) [inline plugins/connections/ui/src/main/java/org/pentaho/di/connections/ui/tree/ConnectionFolderProvider.java:117]: Not sure about letting the project concept in here..
Perhaps an identifier could come from the bowl for project and default?

**Review by tgf — APPROVED** [review]

File: plugins/connections/ui/src/main/java/org/pentaho/di/connections/ui/tree/ConnectionFolderProvider.java (hunk 1 above)
Comment (abryant-hv) [reply plugins/connections/ui/src/main/java/org/pentaho/di/connections/ui/tree/ConnectionFolderProvider.java:117]: coming from the bowl would only work for Global and Project... wouldn't work for Local. 
Also, I'm assuming we'll need to localize these. hrm. I should probably have done that already. I'll open a subtask for that.

File: ui/src/main/java/org/pentaho/di/ui/spoon/tree/TreeManager.java [hunk 2]
```diff
@@ -75,20 +77,26 @@ public void addRoot( String label, List<TreeFolderProvider> providers ) {
...
+    rootNode.addProvider( treeFolderProvider );
   }
 
   public void clear() {
     if ( tree != null ) {
       tree.removeAll();
     }
     treeNodeItemMap.clear();
-    hideAll();
+    //hideAll();
```
Comment (pdesai16) [inline ui/src/main/java/org/pentaho/di/ui/spoon/tree/TreeManager.java:99]: remove the commented code

File: ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java [hunk 3]
```diff
@@ -1177,13 +1180,7 @@ public void updateTreeForActiveAbstractMetas() {
...
       Object managedObject = entry.getObject().getManagedObject();
       if ( managedObject instanceof AbstractMeta ) {
-        if ( managedObject instanceof TransMeta ) {
-          selectionTreeManager.create( (AbstractMeta) managedObject, STRING_TRANSFORMATIONS, props.isOnlyActiveFileShownInTree() );
-        }
-        if ( managedObject instanceof JobMeta ) {
-          selectionTreeManager.create( (AbstractMeta) managedObject, STRING_JOBS, props.isOnlyActiveFileShownInTree() );
-        }
-        selectionTreeManager.show( (AbstractMeta) managedObject );
+        // selectionTreeManager.show( (AbstractMeta) managedObject );
```
Comment (pdesai16) [inline ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java:1183]: commented code

Comment (abryant-hv) [reply ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java]: done

**Review by pdesai16 — APPROVED** [review]

**Review by niehusa — APPROVED** [review]

**Pull request merged**